src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/MissingJavadocMethodCheck.java

```diff
@@ -137,6 +171,34 @@
  *   &lt;property name="ignoreMethodNamesRegex" value="^foo.*$"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * public class Test {
+ *   public void test1() {} // violation, method is missing javadoc
+ *   public void foo() {} // ok
+ *   public void foobar() {} // ok
+ * }
+ * </pre>
+ *
+ * <p>
+ * To configure the check for ignoring missing javadoc for accessor methods:
+ * </p>
+ * <pre>
+ * &lt;module name="MissingJavadocMethod"&gt;
+ *   &lt;property name="allowMissingPropertyJavadoc" value="true"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * public class Test {
+ *   private String text;
+ *
+ *   public void test() {} // violation, method is missing javadoc
+ *   public String getText() { return text; } // ok
+ *   public void setText(String text) { this.text = text; } // ok
+ * }
+ * </pre>
```

Comment:
example for `allowedAnnotations` property is missing

Reply:
Added